src/web/server.tsx

```diff
@@ -0,0 +1,66 @@
+import fastifyFormbody from "@fastify/formbody";
+import fastifyStatic from "@fastify/static";
+import websocket from "@fastify/websocket";
+import Fastify from "fastify";
+import * as elements from "typed-html";
+import { WebAutoMercyDisplay, toggleAutoMercy } from "../commands/automercy";
+import { BaseHtml } from "./baseHtml";
+
+export const fastify = Fastify({
+  logger: false,
+});
+
+export function updateSite(message: string) {
+  fastify.websocketServer.clients.forEach((client) => {
+    client.send(message);
+  });
+}
+// Register websocket plugin
+
+// Create pubsub websocket
+const startServer = async () => {
+  await fastify.register(websocket);
+  await fastify.register(fastifyFormbody);
+  // Static files
+  console.log(process.cwd());
+  fastify.register(fastifyStatic, {
+    root: process.cwd() + "/public",
+    prefix: "/public/",
+  });
+
+  fastify.get("/pubsub", { websocket: true }, (_connection) => {});
+
+  fastify.post("/mercytoggle", async (_request, reply) => {
+    toggleAutoMercy();
+    reply.header("Content-Type", "text/html; charset=utf-8");
+    return <WebAutoMercyDisplay />;
+  });
+
+  fastify.get("/", function (_, reply) {
+    // Set html header
+    reply.header("Content-Type", "text/html; charset=utf-8");
+    return (
+      <BaseHtml>
+        <WebAutoMercyDisplay />
+        <button
```

Comment:
This button uses htmx to update the automercy status. When clicked, it will issue a post request to `/mercytoggle` and replace the element with the id: "status" with whatever the response is.  outerHTML means that it will *replace* the div, not replace the *inside* of the div (which is the default)